spec/models/submission_spec.rb

```diff
@@ -16,7 +16,7 @@
     expect(described_class.create.errors[:user_type]).to be_present
   end
 
-  [:download, :render, :run, :test].each do |action|
```

Comment:
After having a deeper look into the code: You are right. It makes no longer sense here.
We need another test for the download functionality, but it no longer shares the routes with render, run and test, therefore it is not needed here any longer.